websites/documentation/src/content/configuration/plugins.mdx

```diff
@@ -5,8 +5,6 @@ beta: false
 
 ## Gatsby Transformer MDX Introspection
 
-<Warning>This Functionality has been deprecated. MDX v2 provides a rich AST that allows to solve the problems in a cleaner way</Warning>
```

Comment:
I would not make a release of this plugin again just because of that readme change.

Reply:
I agree